src/404.fr-CA.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: layouts/base
+title: Page non trouvée
+lang: fr-CA
+translationKey: 404
+permalink: "{% if lang != config.defaultLanguage %}/{{ config.languages[lang].slug }}{% endif %}/404.html"
+---
+La page que vous recherchez n'existe pas. Retournez à la [page d'accueil?](/)
```

Comment:
The link here sends you to the default home page, not the localized version.

Reply:
```suggestion
La page que vous recherchez n'existe pas. Retournez à la [page d'accueil?](/fr/)
```